### ISSUE TITLE
drc_in_isom: outdated API usage of uniDrcBitstreamDecoder_api.h in MPEG-D part 4

### ISSUE THREAD

**Issue by edrthomas** [MPEGGroup/isobmff#13]
compilation error:

```
./src/DRCData.c: In function ‘initDRCData’:
../src/DRCData.c:68:14: error: too few arguments to function ‘initUniDrcBitstreamDec’
     drcErr = initUniDrcBitstreamDec(drcData->hUniDrcBsDecStruct, sampleRate, framesize);
              ^~~~~~~~~~~~~~~~~~~~~~
In file included from ../src/DRCData.h:39:0,
                 from ../src/DRCData.c:24:
../../../external/Part04-Dynamic_Range_Control/trunk/modules/uniDrcModules/uniDrcBitstreamDecoderLib/include/uniDrcBitstreamDecoder_api.h:53:1: note: declared here  initUniDrcBitstreamDec(HANDLE_UNI_DRC_BS_DEC_STRUCT hUniDrcBsDecStruct,
```

**Comment by edrthomas** [comment]
Now Travis also build the audio tool.

As you can see form the build:

```
../src/DRCData.c: In function ‘initDRCData’:
../src/DRCData.c:68:5: error: too few arguments to function ‘initUniDrcBitstreamDec’
     drcErr = initUniDrcBitstreamDec(drcData->hUniDrcBsDecStruct, sampleRate, framesize);
     ^
In file included from ../src/DRCData.h:39:0,
                 from ../src/DRCData.c:24:
../../../external/Part04-Dynamic_Range_Control/trunk/modules/uniDrcModules/uniDrcBitstreamDecoderLib/include/uniDrcBitstreamDecoder_api.h:53:1: note: declared here
 initUniDrcBitstreamDec(HANDLE_UNI_DRC_BS_DEC_STRUCT hUniDrcBsDecStruct,
 ^
make: *** [../src/DRCData.o] Error 1
```

The code that fails i here:

https://github.com/MPEGGroup/isobmff/blob/2f783d192ac597dd18e55d2f5ec8d27f574adced/IsoLib/drc_in_isom/src/DRCData.c#L68

There may be more places to modify for the new API.

**Comment by edrthomas** [comment]
Fixed by 89043d619d2cc9f2f18128334607e8e730a1674c
